zbus/src/address/transport/nonce_tcp.rs

```diff
@@ -0,0 +1,94 @@
+use std::{borrow::Cow, ffi::OsStr};
+
+use super::{
+    percent::{decode_percents_os_str, decode_percents_str, EncOsStr},
+    tcp::TcpFamily,
+    Address, Error, KeyValFmt, KeyValFmtAdd, Result,
+};
+
+/// `nonce-tcp:` D-Bus transport.
+///
+/// <https://dbus.freedesktop.org/doc/dbus-specification.html#transports-nonce-tcp-sockets>
+#[derive(Debug, Default, PartialEq, Eq)]
+pub struct NonceTcp<'a> {
+    host: Option<Cow<'a, str>>,
+    bind: Option<Cow<'a, str>>,
+    port: Option<u16>,
+    family: Option<TcpFamily>,
+    noncefile: Option<Cow<'a, OsStr>>,
+}
+
+impl<'a> NonceTcp<'a> {
+    /// DNS name or IP address.
+    pub fn host(&self) -> Option<&str> {
+        self.host.as_ref().map(|v| v.as_ref())
+    }
+
+    /// Used in a listenable address to configure the interface on which the server will listen:
+    /// either the IP address of one of the local machine's interfaces (most commonly `127.0.0.1`),
+    /// or a DNS name that resolves to one of those IP addresses, or `*` to listen on all interfaces
+    /// simultaneously.
```

Comment:
Thanks for adding detailed docs. :+1: Not necessarily a blocker but if you could change the docs to follow the convention of:

```
A brief description.

Details in a new paragraph.
```

That would be great.